### ISSUE TITLE
Only works with windows (pypiwin32)

### ISSUE THREAD

**Issue by benknight135** [benknight135/wdd-asylum-claims#7]
For the reading of word documents, the module pypiwin32 is currently required. A better solution must exist somewhere however there doesn't seem to be much support for '.doc' files in Python (as opposed to '.docx' which has tons of support.

- [ ] Find better module for reading '.doc'

- [ ] Find a way to read data from doc without downloading and saving it to a temporary directory

**Comment by dafuloth** [comment]
Yeah it's annoying. DOCX is better because it's just a ZIP archive of XML files.

If we don't want the solution to be Windows-only, the most straightforward (but hacky) workaround I can think of is to implement a linux version of the method, and then have the program detect the platform it's running on so that it calls the appropriate version of the method that relies on `pypiwin32`

I looked into DOC to DOCX file conversion but looks to me like that runs into exactly the same problems - no universal platform-independent way to work with DOC files. [StackOverflow: multiple .doc to .docx file conversion using python](https://stackoverflow.com/questions/38468442/multiple-doc-to-docx-file-conversion-using-python)

As for point 2: If I understand you correctly, what you propose is that the file should just be retrieved and processed in memory - without the overhead of writing to a file on disk and then reading the file?
